README.md

```diff
@@ -475,38 +475,69 @@ Where `<world>` is the name of the world you wish to map (omit for all
 worlds).
 
 ## Mapping the world
-Minecraft Server Control Script uses [Overviewer]
-(http://docs.overviewer.org/en/latest/) to generate maps. After [installing]
-(#mapping-software), modify the settings (if necessary) found in the
-`mscs.defaults` file (see [adjusting global server properties]
-(#adjusting-global-server-properties)):
+Minecraft Server Control Script uses [Overviewer](http://docs.overviewer.org/en/latest/) to generate maps. 
+
+### Installing Overviewer
+
+Overviewer has pretty straightforward documentation to download and install the software:
+
+* [Debian/Ubuntu](http://overviewer.org/debian/info)
+* [RHEL/CentOS/Fedora](http://overviewer.org/rpms/info)
+
+*Once you follow the install page, come back here for further instructions.
```

Comment:
Should we use blockquotes for notes like this? They stand out a little bit better when the markdown is rendered, but they do look odd when reading the raw file.

```
> __NOTE:__ Once you follow the install page...
```

> __NOTE:__ Once you follow the install page...